src/main/java/net/ccbluex/liquidbounce/injection/mixins/minecraft/network/MixinClientConnection.java

```diff
@@ -54,6 +61,10 @@ protected static <T extends PacketListener> void handlePacket(Packet<T> packet,
     private void hookSendingPacket(Packet<?> packet, final CallbackInfo callbackInfo) {
         final PacketEvent event = new PacketEvent(TransferOrigin.SEND, packet, true);
 
+        if (packet instanceof PlayerMoveC2SPacket) {
+            this.sendInternal(new ClientTickEndC2SPacket(), null, true);
```

Comment:
we're flagging timer with this one :fire:

Reply:
I also think scripts should have the control to not send the packet.